packages/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -41,6 +41,15 @@ const columnHeaderStyles = {
   },
 };
 
+declare module '@mui/material' {
+  interface Mixins {
+    DataGrid: {
+      containerBackground?: string;
+      pinnedBackground?: string;
+    };
+  }
+}
```

Comment:
Really not sure if this is the right place to add it, need feedback.

Reply:
I think we used to add augmentations in `overrides.ts` like:
https://github.com/mui/mui-x/blob/3d86411074f2fe291d42edfa4b930f9baa951f62/packages/x-data-grid/src/themeAugmentation/overrides.ts#L7-L9

Also, should we target `@mui/material/styles` instead?
```diff
- declare module '@mui/material' {
+ declare module '@mui/materia/styles' {
```

Reply:
I have no idea. I've changed it for now.

Reply:
I have moved them to `overrides.ts` but the TS check CI fails. When/how should those augmentations be imported?

Reply:
I have added an import to theme augmentations in `GridRootStyle.ts` to make it work.